explorer/app/components/Project/components/Citation/citation.tsx

```diff
@@ -0,0 +1,49 @@
+// Core dependencies
+import { Typography } from "@mui/material";
+import React from "react";
+
+// App dependencies
+import { useConfig } from "app/hooks/useConfig";
+import { Stack } from "../../../common/Stack/Stack";
+import { ANCHOR_TARGET, Link } from "../../../Links/components/Link/link";
+import { Section } from "../Section/section";
+
+// Styles
+import { CitationLink } from "./citation.styles";
+
+interface Props {
+  citationPath: string;
+}
+
+export const Citation = ({ citationPath }: Props): JSX.Element => {
+  const { datasources } = useConfig();
+  const { url } = datasources;
```

Comment:
It looks like we need to add an additional URL to the config to generate the citation URL. The `{url}` pulled from datasources is the API endpoint but we need the Explorer URL. Take a look at the Angular config to see how we generate it there.